mockall/src/lib.rs

```diff
@@ -1602,10 +1610,23 @@ impl Times {
         (self.range.0.end - self.range.0.start) == 1
     }
 
-    /// Has this expectation already been called the minimum required number of
-    /// times?
-    pub fn is_satisfied(&self) -> bool {
-        self.count.load(Ordering::Relaxed) >= self.range.0.start
+    /// Has this expectation already been called a number of times above the minimum,
+    /// below the maximum, both cases, or neither?
```

Comment:
Let me suggest some more natural English:
```suggestion
    /// Has this expectation already been called the expected number of times?
    /// If not, was it too many or too few?
```